src/Tribe/Template.php

```diff
@@ -613,4 +613,55 @@ public function template( $name, $context = array(), $echo = true ) {
 
 		return $html;
 	}
+
+	/**
+	 * Sets a number of values at the same time.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $values An associative key/value array of the values to set.
+	 * @param bool  $is_local Whether to set the values as global or local; defaults to local as the `set` method does.
+	 *
+	 * @see Tribe__Template::set()
+	 */
+	public function set_values( array $values = [], $is_local = true ) {
+		foreach ( $values as $key => $value ) {
+			$this->set( $key, $value, $is_local );
+		}
+	}
+
+	/**
+	 * Returns the Template global context.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative key/value array of the Template global context.
+	 */
+	public function get_global_values() {
+		return $this->global;
+	}
+
+	/**
+	 * Returns the Template local context.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative key/value array of the Template local context.
+	 */
+	public function get_local_values() {
+		return $this->context;
```

Comment:
Filter local values, and maybe `get_template_local_values()`, not sure if we need it.

Reply:
Since this is a method on the template object calling it `get_template_local_values()` seem redundant:
```php
$template->get_template_local_values(); // redundant
$template->get_local_values(); // cleaner
```

Is there value in filtering the values? Here we're getting the values that have been set on the template and then we read them. What value filtering the values would provide? The Template is not going to use the filtered version, but the unfiltetered one.